cmd/skopeo/inspect.go

```diff
@@ -135,9 +148,19 @@ func (opts *inspectOptions) run(args []string, stdout io.Writer) (retErr error)
 		}, opts.retryOpts); err != nil {
 			return errors.Wrapf(err, "Error reading OCI-formatted configuration data")
 		}
-		err = json.NewEncoder(stdout).Encode(config)
+		if report.IsJSON(opts.format) || opts.format == "" {
+			var out []byte
+			out, err = json.MarshalIndent(config, "", "    ")
+			if err == nil {
+				fmt.Fprintf(stdout, "%s\n", string(out))
+			}
+		} else {
+			row := "{{range . }}" + report.NormalizeFormat(opts.format) + "{{end}}"
```

Comment:
Why is `NormalizeFormat` desirable? The man page says “Go template”, nothing about treating `\t` specially, nor about the `table ` special case.

What does the `{{ range . }}…{{end}}` do that we need the `data` variable instead of just using an `interface{}` for the struct directly? At least https://golang.org/pkg/text/template/#example_Template suggests that should work.

(And if it is important, shouldn’t the `report` package do that?)